server/models/user.js

```diff
@@ -35,6 +35,14 @@ module.exports = (sequelize, DataTypes) => {
         type: STRING,
         allowNull: false
       },
+      passwordRetries: {
+        type: INTEGER,
+        allowNull: false
+      },
+      lastLocked: {
```

Comment:
Semantically lastLocked is actually the time it unlocks right? I think either it'd be called unlockTime or something, or the value of lastLocked would be the time that it actually locks and then you check that the current time is 10 minutes after lastLocked. This is a minor thing though.